src/source/git_source.rs

```diff
@@ -390,30 +393,11 @@ mod tests {
                 ),
                 "rattler-build",
             ),
-            (
-                GitSource::create(
-                    GitUrl::Path("../rattler-build".parse().unwrap()),
-                    GitRev::default(),
-                    None,
-                    vec![],
-                    None,
-                    false,
-                ),
-                "rattler-build",
-            ),
         ];
         let system_tools = crate::system_tools::SystemTools::new();
 
         for (source, repo_name) in cases {
-            let res = git_src(
-                &system_tools,
-                &source,
-                cache_dir.as_ref(),
-                // TODO: this test assumes current dir is the root folder of the project which may
-                // not be necessary for local runs.
-                std::env::current_dir().unwrap().as_ref(),
```

Comment:
You can use the `CARGO_MANIFEST_DIR` perhaps?

https://doc.rust-lang.org/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-crates

Reply:
ups, sorry, just realized that you _removed_ this code :D